### PR TITLE
Add NVIDIA RTX 2070 to hardware.ts list

### PR DESCRIPTION
**Pull request by WarpWing**
Just came across this: https://huggingface.co/settings/local-apps

Noticed it didn't have the NVIDIA RTX 2070 on it so this PR aims to add it. I'm going off of FP32 TFLOPS performance.
image
https://www.techpowerup.com/gpu-specs/geforce-rtx-2070.c3252

### CODE REVIEW

**Review by julien-c — COMMENTED** [review]
w00t

File: packages/tasks/src/hardware.ts [hunk 1]
```diff
@@ -159,7 +159,11 @@ export const SKUS = {
 			"RTX 3060": {
 				tflops: 12.74,
 				memory: [12, 8],
-			},
+			},
```
Comment (pcuenca) [inline packages/tasks/src/hardware.ts:162]:
```suggestion
			},
```

Just removing a trailing space :)

File: packages/tasks/src/hardware.ts [hunk 2]
```diff
@@ -159,7 +159,11 @@ export const SKUS = {
 			"RTX 3060": {
 				tflops: 12.74,
 				memory: [12, 8],
-			},
+			}, 
+			"RTX 2070": {
+				tflops: 7.465,
```
Comment (pcuenca) [inline packages/tasks/src/hardware.ts:164]:
```suggestion
				tflops: 14.93,
```

Actually, we use the float16 numbers for reference :)

**Review by pcuenca — COMMENTED** [review]
Awesome!

File: packages/tasks/src/hardware.ts (hunk 2 above)
Comment (WarpWing) [reply packages/tasks/src/hardware.ts:164]: > Actually, we use the float16 numbers for reference :)

Ahhhh, didn't know if it was FP16 or FP32.

**Comment by pcuenca** [comment]
Thank you @WarpWing!

**Pull request merged**